src/extrovermectin.ts

```diff
@@ -44,7 +44,7 @@ export function expectedGregs(): number {
 }
 
 export function doingExtrovermectin(): boolean {
-  return get("beGregariousCharges") > 0;
+  return true; // get("beGregariousCharges") > 0;
```

Comment:
...

Reply:
Some of the free runs compete with extro for resources, and so we have to do the free runs after we compute diet. We want to compute the diet after we do most free fights so that way we can safely compute what our Meat Drop modifier is. 
I marked this one as true simply because I was under the assumption that everyone testing this would always be doing extro, since that is a significantly better usage of resources